File: EnKucukDeger.cs
using System;

class Program
{
    static int Min(int[] x)
    {
        int enKucuk = x[0];

        for (int i = 1; i < x.Length; ++i)
        {
            if (x[i] < enKucuk)
            {
                enKucuk = x[i];
            }
        }

        return enKucuk;
    }

    static void Main()
    {
        int s1, s2, s3;

        Console.Write("1. sayı : ");
        s1 = Convert.ToInt32(Console.ReadLine());

        Console.Write("2. sayı : ");
        s2 = Convert.ToInt32(Console.ReadLine());

        Console.Write("3. sayı : ");
        s3 = Convert.ToInt32(Console.ReadLine());

        int[] liste = { s1, s2, s3 };

        int result = Min(liste);
        Console.WriteLine("En küçük sayı: " + result);
    }
}

File: EnKucukDeger.py
def min_(x):
    enKucuk = x[0]

    for eleman in x:
        if eleman < enKucuk:
            enKucuk = eleman

    return enKucuk

s1 = int(input("1. sayı: "))
s2 = int(input("2. sayı: "))
s3 = int(input("3. sayı: "))

liste = [s1, s2, s3]

result = min_(liste)
print(result)
